plugins/jobs/src/js/components/config/DependenciesConfigSection.tsx

```diff
@@ -8,7 +8,7 @@ import { Link } from "react-router";
 
 export default class DependenciesConfigSection extends BaseConfig<JobOutput> {
   shouldExcludeItem(_: Value<JobOutput>) {
-    return false && !!this.props.config.dependencies?.length;
+    return !this.props.config.dependencies?.length;
```

Comment:
Do we have a test for it?

Reply:
This type coercion is not obvious for my. Why removing `false &&` fixes the issues?

Reply:
Oh, now I see it :grey_exclamation::exclamation: